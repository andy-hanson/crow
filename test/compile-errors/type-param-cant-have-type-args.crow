main nat64^(_ string[])
	()

f[t] void(_ nat t)
	()
