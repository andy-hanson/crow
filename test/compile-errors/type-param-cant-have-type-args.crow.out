exit code: 0
stderr:
test/compile-errors/type-param-cant-have-type-args.crow 4:17-4:18 a type parameter can't take type arguments
